### PR TITLE
feat(global): add global variable that allow to create market

### PR DESCRIPTION
**Pull request by shmodarresi**

### CODE REVIEW

File: src/router/routes.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ if (WERT_PARTNER_ID) {
   });
 }
 
-if (ENABLE_MARKET_CREATION) {
+if (ENABLE_MARKET_CREATION || MARKET_CRREATOR) {
```
Comment (shekhar-shubhendu) [inline src/router/routes.ts]: typo: `MARKET_CRREATOR` => `MARKET_CREATOR `

**Review by shekhar-shubhendu — DISMISSED** [review]

File: src/pages/MainPage/MainPage.tsx [hunk 2]
```diff
@@ -186,7 +187,9 @@ export const MainPage: React.FC<MainPageProps> = ({ title, children, description
             actionText={t('signOut')}
             userBalance={balance}
             primaryActionText={t('connectWallet')}
-            secondaryActionText={ENABLE_MARKET_CREATION ? t('createQuestionPage') : undefined}
+            secondaryActionText={
+              ENABLE_MARKET_CREATION || MARKET_CREATOR ? t('createQuestionPage') : undefined
```
Comment (tlebon) [inline src/pages/MainPage/MainPage.tsx]: doesnt the MARKET_CREATOR need to equal the current address? or else i think this doesn't work as intended

File: src/router/routes.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ if (WERT_PARTNER_ID) {
   });
 }
 
-if (ENABLE_MARKET_CREATION) {
+if (ENABLE_MARKET_CREATION || MARKET_CREATOR) {
```
Comment (tlebon) [inline src/router/routes.ts:27]: same here, MARKET_CREATOR should equal the logged in user for the admin to be able to create the market ( again to my understanding)

**Review by tlebon — CHANGES_REQUESTED** [review]
check my comments

File: src/router/routes.ts (hunk 3 above)
Comment (shekhar-shubhendu) [reply src/router/routes.ts:27]: not for here. we want to exclude the page at the build time itself.

File: src/pages/MainPage/MainPage.tsx (hunk 2 above)
Comment (shekhar-shubhendu) [reply src/pages/MainPage/MainPage.tsx]: yes. right.

Comment (shekhar-shubhendu) [reply src/pages/MainPage/MainPage.tsx]:
```
ENABLE_MARKET_CREATION || (activeAccount?.address && MARKET_CREATOR && MARKET_CREATOR === activeAccount?.address)
```

File: src/router/routes.ts (hunk 3 above)
Comment (tlebon) [reply src/router/routes.ts:27]: ah correct

Comment (tlebon) [reply src/router/routes.ts:27]: so ignore this comment

**Review by tlebon — APPROVED** [review]

**Pull request merged**